### DISCUSSION TITLE
Calendar and Kanban views are missing

### DISCUSSION THREAD

**Discussion by yobotn** [nocodb/nocodb#660] [Features]
Dear NoCoDB Team,

I have nocodb installation version 11.46, im using mysql and im missing many features likes the Calendar View, Kanban View, Share View and many other features.

what i am missing ?

Thank you

**Comment by T100D** [comment]
Same problem on version 81.1.

**Comment by mrn3** [comment]
Yeah this seems to still be a problem for me on
"nocodb": "^0.81.1"

![image](https://user-images.githubusercontent.com/3468214/140788594-b8e4eb96-eda8-4818-9aa8-04ad67e8e587.png)

**Comment by sangletech** [comment]
I got told by the product team that the Calendar and Kanban views are not officially released yet.

**Comment by wingkwong** [comment]
Calendar View and Kanban View are still in the roadmap while Share View is available.
